server/network/aoprotocol.py

```diff
@@ -2078,6 +2103,10 @@ def net_cmd_zz(self, args):
         if not self.client.is_checked:
             return
 
+        if self.client.server.config["whitelist"] and not self.client.is_wlisted:
+            self.client.send_ooc('You must be whitelisted from discord!')
```

Comment:
This and every other instnace needs to tell the user the exact steps to take to whitelist themselves. Here's my suggestion:
```suggestion
self.client.send_ooc('You need to verify your connection with a Discord login! Use /whitelist your_discord_name and follow instructions to do so.')
```